<launch>
<!-- Turtlesim Node-->
  <node pkg="turtlesim" type="turtlesim_node" name="sim"/>

 <!-- Axes -->
  <param name="axis_linear" value="1" type="int"/>
  <param name="axis_angular" value="0" type="int"/>
  <param name="scale_linear" value="2" type="double"/>
  <param name="scale_angular" value="2" type="double"/>
  <node pkg="turtle_control" type="turtle_ar.py" name="turtle_ar"/>

<arg name="marker_size" default="6" />
	<arg name="max_new_marker_error" default="0.08" />
	<arg name="max_track_error" default="0.2" />
	<arg name="cam_image_topic" default="/usb_cam/image_rect_color" />
	<arg name="cam_info_topic" default="/usb_cam/camera_info" />	
	<arg name="output_frame" default="/marker" />

	<node name="ar_track_alvar" pkg="ar_track_alvar" type="individualMarkersNoKinect" respawn="false" output="screen" args="$(arg marker_size) $(arg max_new_marker_error) $(arg max_track_error) $(arg cam_image_topic) $(arg cam_info_topic) $(arg output_frame)" />e>

	<node ns="usb_cam" pkg="image_proc" type="image_proc" name="image_proc" />
	<node pkg="usb_cam" type="usb_cam_node" name="usb_cam" output="screen">
		<param name="width" type="int" value="640" />
		<param name="height" type="int" value="480" />
		<param name="fps" type="int" value="30" />
		<param name="frame_id" type="string" value="/head_camera" />
		<param name="device" type="string" value="/dev/video1" />
	</node>

	<node pkg="tf" type="static_transform_publisher" name="marker_to_camera" output="screen" args="0 0 0 0.785398163 0 0 marker head_camera 10" />

	<!--<node pkg="rviz" type="rviz" name="rviz" />-->


</launch>


